client/signup/steps/jpo-summary/style.scss

```diff
@@ -0,0 +1,61 @@
+.jpo__summary-wrapper {
+	text-align: center;
+}
+
+.jpo__summary-col-header {
+	font-weight: bold;
+}
+
+.jpo__summary-wrapper table {
+	max-width: 480px;
+
+	@include breakpoint( "<480px" ) {
+		width: 100%;
+		margin: 0 10px;
+	}
+
+	@include breakpoint( ">480px" ) {
+		margin-left: auto;
+		margin-right: auto;
+	}
+}
+
+.jpo__summary-wrapper table td {
```

Comment:
Do we need a table for this? I am unable to see this particular view yet because of connection errors (will fix soon), but if we don't need a table, let's avoid it. That way we can avoid these nested selectors.